internal/fs/fs.go

```diff
@@ -337,3 +360,13 @@ func IsRegular(name string) (bool, error) {
 	}
 	return true, nil
 }
+
+// IsSymlink determines if the given path is a symbolic link.
```

Comment:
The reason why they are skipped in the tests is that creating a new symlink is not supported in Go. But if the user already created one, it will be detected by this function.